src/app/researcher/study/request/[orgSlug]/study-proposal.tsx

```diff
@@ -154,6 +154,7 @@ export const StudyProposal: React.FC<{ orgSlug: string }> = ({ orgSlug }) => {
                 color: 'green',
             })
             router.push(`/researcher/dashboard`)
+            window.location.assign(`/researcher/dashboard`)
```

Comment:
???  I don't think we want to do this, but if we did, we should remove the router.push before it